### ISSUE TITLE
Failed pulling filesystem: mkdir /tmp/terraform-build-worker···/fs-download/···: not a directory

### ISSUE THREAD

**Issue by 0x2b3bfa0** [hashicorp/terraform#25119]
### Terraform Version
```
0.12.26
```

### Terraform Configuration Files
> Attached to the issue: [code.tar.gz](https://github.com/hashicorp/terraform/files/4725732/code.tar.gz)

### Debug Output
> Attached to the issue: [terraform-plan.txt](https://github.com/hashicorp/terraform/files/4725722/terraform-plan.txt)

### Expected Behavior
> Successful execution

### Actual Behavior
```
failed pulling filesystem: mkdir /tmp/terraform-build-worker···/fs-download/···: not a directory
```

### Steps to Reproduce
```bash
terraform init
terraform plan
```

### Additional Context
> This issue only affects Terraform Cloud plans; local plans run successfully.

**Comment by 0x2b3bfa0** [comment]
I couldn't find the `pulling filesystem` anywhere, so this bug might be related with some private container component.

**Comment by DoubleLoopAlex** [comment]
I'm also getting this issue with TF Cloud. Any updates? Did you fix?

**Comment by 0x2b3bfa0** [comment]
@DoubleLoopAlex, no, I didn't. My workaround was to use `snake_case` paths instead.
